### PR TITLE
Added vapi, binding C library to vala language

### PR DESCRIPTION
**Pull request by zzazang**
I added vapi file to bind this library with vala language
- added _create function who returns itself

### CODE REVIEW

**Comment by benmcollins** [comment]
Hello. Thanks for your pull request. I have a couple of things that need to be done before I can accept it:

1) You added code that is using 2-space indentation, and the entire rest of my code is using 1-tab for indentation. Please make your indentation match the rest of the code.
2) Your commit summary (the first line of the commit) is very short. The summary should be short, but generally should be a complete sentence.
3) Please use `git commit -s ...` for your commits so a full `Signed-off-by:` line is included with your name and email address.

Thanks!

**Comment by zzazang** [comment]
Hello
modified as you asked. please review the commits!

File: include/jwt.h [hunk 1]
```diff
@@ -49,6 +49,18 @@ typedef enum jwt_alg {
...
+ * Allocate and return a new, empty, JWT object.
+ *
+ * This is used to create a new object for a JWT.
+ * Using jwt_new, returns new allocated jwt object.
+ * with the returned object, use jwt_free() to clean up the memory used by it.
+ *
+ * @return jwt Pointer to a JWT object pointer. Will be allocated on
+ *     success. Null on fail.
+ */
+jwt_t *jwt_create();
```
Comment (benmcollins) [inline include/jwt.h]: Please complete the declaration with a `void`

File: libjwt/jwt.c [hunk 2]
```diff
@@ -1,19 +1,19 @@
 /* Copyright (C) 2015 Ben Collins <ben@cyphre.com>
```
Comment (benmcollins) [inline libjwt/jwt.c]: Please remove all white space changes from this commit.

**Comment by benmcollins** [comment]
No updates to rectify issues in this commit and it has conflicts with master. Closing.

**Pull request closed without merging**